ee/clickhouse/queries/experiments/trend_experiment_result.py

```diff
@@ -61,52 +64,83 @@ def __init__(
 
     def get_results(self):
         insight_results = self.insight.run(self.query_filter, self.team)
-        variants = self.get_variants(insight_results)
+        control_variant, test_variants = self.get_variants(insight_results)
+
+        probabilities = self.calculate_results(control_variant, test_variants)
 
-        probability = self.calculate_results(variants)
+        mapping = {
+            variant.name: probability for variant, probability in zip([control_variant, *test_variants], probabilities)
```

Comment:
this should be variant key

Reply:
the variant here is actually defined on L18: https://github.com/PostHog/posthog/pull/7924/files#diff-709ec28722b679d006e06a7f4ded9ca56468eaa42c7fde8217f70a6ab7ffbea7R19

key is the name for the Experiment 😅 . These are separate from the variants defined for the FF.

Although I can see where the confusion comes in, will rename this to key instead!

Reply:
sweet thanks!